src/translation.py

```diff
@@ -7,17 +7,20 @@ class Translation:
     def __init__(self, reference: Sentence = None, hypothesis: Sentence = None):
         self.r = reference
         self.h = hypothesis
+        self.errors = None
 
     def count_all_lemmas(self):
         lemmas = Counter(
-            [word.lemma for word in self.r.words if word.lemma != "_"] +
-            [word.lemma for word in self.h.words if word.lemma != "_"]
+            [word.lemma for word in self.r.words if word.lemma] +
+            [word.lemma for word in self.h.words if word.lemma]
         )
         return lemmas
 
     def remove_correct_translation(self):
-        for r_word in self.r.words:
-            for h_word in self.h.words:
-                if r_word == h_word:
-                    self.h.words.remove(h_word)
-                    self.r.words.remove(r_word)
+        potential_r_errors = [word for word in self.r.words if word not in self.h.words]
```

Comment:
this is problematic because if
ref = "dog dog"
hyp = "dog dogs"
you will remove both of the "dog" in the ref instead just one